validators/material.validator.js

```diff
@@ -0,0 +1,28 @@
+const Joi = require('joi');
+
+const {
+  materialPurposes: { MAIN, INNER, BOTTOM, PATTERN, CLOSURE },
+} = require('../consts/material-purposes');
+
+const languageSchemaValidator = {
```

Comment:
please change languageSchemaValidator into a function, that will take min and max as arguments. It should return joi schema.